microsoft-365/lti/teams-meetings-with-canvas.md

```diff
@@ -74,3 +74,9 @@ As a Canvas Admin, you'll need to add the Microsoft Teams meetings LTI app withi
 5. Select **Install**.
 
    The Microsoft Teams meetings LTI app will be added to the list of external apps.
+   
+## Enable for Canvas Courses
+
+In order to use the LTI within a course, an instructor of the Canvas course must enable the integrations sync. Each course must be enabled by an instructor for a corresponding Teams to be created; there is no global mechanism for Teams creation. This is designed out of caution to prevent unwanted Teams being created.
```

Comment:
```suggestion
To use the LTI app within a course, an instructor of the Canvas course must enable integrations sync. Each course must be enabled by an instructor for a corresponding team to be created; there is no global mechanism for teams creation. This is designed as a precautionary measure to prevent unwanted teams from being created.
```